CMSgov/mint-app
# [Easi-2057] Front End for Payments

# EASI-2057

## Changes and Description

Use Figma for a reference https://www.figma.com/file/M72DReE73SSXMN48WZfaKN/MINT---New-Model-Process-(Model-Intake)?node-id=1210%3A150733 

- Add all the Payment sections
- Add tests



## How to test this change

- Navigate to Task List of a model plan
- Find that `Payment` is "Start"-able
- Ensure all pages match the figma



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
